quality-assurance/safety-assurance-guidance.md

```diff
@@ -1,44 +1,47 @@
-* Let the PO make final decisions on the safety impact/consequence. The PO will engage with other SME's to determine if the safety risk is acceptable. This may  
-  mean that although there is a potential safety risk that risk is deemed manageable or may require remedial action. This detail should be captured in a lightweight 
-  manner within the relevant PBI.
- 
-## General Principles 
+1. If a PBI is considered as potentially relevant to safety, it must be marked in the work tracking system with the standard tag ‘Safety Pending’.
+2. The development team must then discuss safety requirements for that PBI.  This discussion may be at any team event (e.g. refinement, threat-modelling, or a specific safety meeting).
+3. Under the direction of the test engineer, the team must discuss whether a failure condition relating to the PBI can cause an error or delay in the product.  If it is decided that this is the case, the PBI must be marked with the standard tag ‘Safety’.  If it is not the case, then all safety tags may be removed.
+4. The team treats the potential hazard as a mini bowtie analysis, considering i) Barriers to the failure condition happening ii)	Mitigations to prevent the failure condition creating error or delay
+5. These barriers and mitigations are expressed as acceptance criteria on the PBI.  As a result, their implementation, testing  and move to production will be traceable in the work tracking tool.
 
-* The safety of a system can only be determined in the context of the whole (all of the component parts).
-* Reliable software does not equal safe software.
-* Safety Modelling using the Bowtie method is carried out at the right level (project level normally gives the best return).
-* Software with safety related functionality must have clear evidence that all safety requirements have been met or relevant mitigations have been agreed.
-* Failure modes, including hardware, software, human and system are addressed in the design of the software.
-* The test engineers in the team have accountability for making sure safety is considered as part of their overall remit but the whole team have responsibility for inputting into the process.
-* Reliance on manual processes is minimised.
-* Best practices in software and test engineering are applied.
-* Sound human engineering principles are applied to the design of the software user interface to minimise the probability of human error.
-* Speak up, if you feel that safety is being compromised in some way, raise it!
-* Exercise judgement, not the same as going out on a limb but exercising common sense.
+
+# Accountability for security modelling
+
+The team’s Test Engineer is accountable for this process being followed.
+
+# Skills
+
+All participants must have a basic understanding of bow-tie risk analysis.  Training material will be provided by the Test Practice.
```

Comment:
Be consistent with bowtie, Bowtie or bow-tie on the page.

Reply:
settled on bow tie